src/shared/label.c

```diff
@@ -25,9 +25,11 @@
 #include "label.h"
 
 int label_fix(const char *path, bool ignore_enoent, bool ignore_erofs) {
-        int r, q;
+        int r = 0, q = 0;
```

Comment:
Assignment to q will always create an unused assigned value warning, as q is always assigned to by mac_smack_fix() below.